src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -19,6 +19,77 @@ See License.txt for details.
 //----------------------------------------------------------------------------
 vtkStandardNewMacro(vtkPlusWinProbeVideoSource);
 
+  //----------------------------------------------------------------------------
+  // Define command strings
+const char* vtkPlusWinProbeVideoSource::SET_FREEZE                   = "SetFrozen";
+const char* vtkPlusWinProbeVideoSource::GET_FREEZE                   = "GetFrozen";
+const char* vtkPlusWinProbeVideoSource::SET_TGC                      = "SetTGC";
+const char* vtkPlusWinProbeVideoSource::GET_TGC                      = "GetTGC";
+const char* vtkPlusWinProbeVideoSource::SET_ALL_FOCAL_DEPTHS         = "SetAllFocalDepths";
+const char* vtkPlusWinProbeVideoSource::SET_FOCAL_DEPTH              = "SetFocalDepth";
+const char* vtkPlusWinProbeVideoSource::GET_FOCAL_DEPTH              = "GetFocalDepth";
+const char* vtkPlusWinProbeVideoSource::SET_ALL_ARFI_FOCAL_DEPTHS    = "SetAllARFIFocalDepths";
+const char* vtkPlusWinProbeVideoSource::SET_ARFI_FOCAL_DEPTHS        = "SetARFIFocalDepth";
+const char* vtkPlusWinProbeVideoSource::GET_ARFI_FOCAL_DEPTHS        = "GetARFIFocalDepth";
+const char* vtkPlusWinProbeVideoSource::SET_B_MULTIFOCAL_ZONE_COUNT  = "SetBMultifocalZoneCount";
+const char* vtkPlusWinProbeVideoSource::GET_B_MULTIFOCAL_ZONE_COUNT  = "GetBMultifocalZoneCount";
+const char* vtkPlusWinProbeVideoSource::SET_FIRST_GAIN_VALUE         = "SetFirstGainValue";
+const char* vtkPlusWinProbeVideoSource::GET_FIRST_GAIN_VALUE         = "GetFirstGainValue";
+const char* vtkPlusWinProbeVideoSource::SET_TGC_OVERALL_GAIN         = "SetTGCOverallGain";
+const char* vtkPlusWinProbeVideoSource::GET_TGC_OVERALL_GAIN         = "GetTGCOverallGain";
+const char* vtkPlusWinProbeVideoSource::SET_SPATIAL_COMPOUND_ENABLED = "SetSpatialCompoundEnabled";
+const char* vtkPlusWinProbeVideoSource::GET_SPATIAL_COMPOUND_ENABLED = "GetSpatialCompoundEnabled";
+const char* vtkPlusWinProbeVideoSource::GET_SPATIAL_COMPOUND_ANGLE   = "GetSpatialCompoundAngle";
+const char* vtkPlusWinProbeVideoSource::SET_SPATIAL_COMPOUND_COUNT   = "SetSpatialCompoundCount";
+const char* vtkPlusWinProbeVideoSource::GET_SPATIAL_COMPOUND_COUNT   = "GetSpatialCompoundCount";
+const char* vtkPlusWinProbeVideoSource::SET_MMODE_ENABLED            = "SetMModeEnabled";
+const char* vtkPlusWinProbeVideoSource::GET_MMODE_ENABLED            = "GetMModeEnabled";
+const char* vtkPlusWinProbeVideoSource::SET_M_REVOLVING_ENABLED      = "SetMRevolvingEnabled";
```

Comment:
Looks like a missed command for `GetMRevolvingEnabled`

Reply:
updated